app/models/cloud_network.rb

```diff
@@ -41,11 +41,7 @@ class CloudNetwork < ApplicationRecord
 
   def self.class_by_ems(ext_management_system, external = false)
```

Comment:
If `external` is now ignored here, can you change the parameter name here to `_external`?